.github/workflows/ci.yml

```diff
@@ -9,5 +9,7 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - name: Run Lint Tests
-        run: echo "Hello, world!"
\ No newline at end of file
+      - name: flake8
+        run: pip install flake8
+        uses: py-actions/flake8@v2
+        run: flake8 --max-line-length=99 . || true
```

Comment:
Not sure if you already saw it, but this doesn't seem to be working: https://github.com/LucyJimenez/asv/actions/runs/1770088885

Also, can you move the flake8 settings to `setup.cfg` as we discussed please.

Reply:
Yes, I saw it. So, I tried something different and also added the setup.cfg. Please, check the changes.